store.go

```diff
@@ -1145,7 +1264,8 @@ func (s *store) canUseShifting(uidmap, gidmap []idtools.IDMap) bool {
 
 func (s *store) PutLayer(id, parent string, names []string, mountLabel string, writeable bool, options *LayerOptions, diff io.Reader) (*Layer, int64, error) {
 	var parentLayer *Layer
-	rlstore, err := s.getLayerStore()
+	// We always put layers to the persistent store (rlstore), as container layers are not put
```

Comment:
Do you mean
```suggestion
	// We always put layers to the persistent store (rlstore): container layers are not created from a diff tarball,
	// so this is known to be a non-container layer.
```
?

But that’s not actually required by the implementation AFAICS; it is possible to call this with `writeable = true, diff = nil`.